diff --git a/src/contents/ui/mobile.qml b/src/contents/ui/mobile.qml
index 98778b5..c5a7ef8 100644
--- a/src/contents/ui/mobile.qml
+++ b/src/contents/ui/mobile.qml
@@ -99,7 +99,7 @@ Kirigami.ApplicationWindow {
                 text: i18n("Settings")
                 onTriggered: {
                     popSubPages();
-                    pageStack.push("qrc:/SettingsPage.qml");
+                    pageStack.pushDialogLayer("qrc:/SettingsPage.qml");
                 }
             }
         ]

