From f400077864908e6d8c67322316d098f7c43a903f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonah=20Br=C3=BCchert?= <jbb@kaidan.im>
Date: Thu, 6 Nov 2025 01:10:27 +0100
Subject: [PATCH] Update sdk to 6.10

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ef7c833..ebb1b6d6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,6 +62,11 @@ find_package(KF6 ${KF_MIN_VERSION} REQUIRED COMPONENTS Kirigami Purpose I18n Con
 find_package(KF6KirigamiAddons 0.6 REQUIRED)
 set_package_properties(KF6QQC2DesktopStyle PROPERTIES TYPE RUNTIME)
 
+# TODO: This should probably be done in QCoros installed CMake file instead.
+if(Qt6Core_VERSION VERSION_GREATER_EQUAL "6.9.0")
+    find_package(Qt6 ${REQUIRED_QT_VERSION} CONFIG REQUIRED QmlPrivate)
+endif()
+
 # Necessary to support QtWebEngine installed in a different prefix than the rest of Qt (e.g flatpak)
 find_package(Qt6WebEngineCore REQUIRED)
 find_package(Qt6WebEngineQuick REQUIRED)
diff --git a/org.kde.angelfish.json b/org.kde.angelfish.json
